scripts/fuse/README.MD

```diff
@@ -0,0 +1,166 @@
+E.g. the existence of `user:/a` and `user:/a/b/c` does not imply the existence of `user:/a/b`.
+Such intermediate keys are mapped to "virtual" directories, s.t. navigation in an interactive manner is possible and the directory tree is continous.
+
+A side effect is that creating/deleting a single key can create/delete multiple "virtual" directories at once.
+Another is that copies created with `cp -r` will not have these vacuous intermediaries but instead empty keys.
+
+### Keys that act both as file and directory
+
+In Elektra, there are only keys and no distinction between directories and files are made. To enable a bidirectional mapping into a classical file system, the following mechanism is employed:
+
+If a key (`A`) exists, and has some child key (`B`), `A` is treated as a directory containing `B` and a special file `@elektra.value`. This file is used to access the value of `A`.
+
+Some side effects arise:
+E.g. recursively deleting directories with `rm -r` containing `@elektra.value` files raises (task uncritical) errors when attempting to delete a parent directory which was already deleted with the removal of an `@elektra.value` file directly below.
+
+### "Leaf"-Keys that act as as directories
+
+Keys at the bottom of the hierarchy are normally treaded as files. However in certain cases, like explicitly creating directories using `mkdir`, this would result in files beeing displayed, making it impossible to expand the hierarchy using standard tools.
+Therefore, when a directory is created, the special meta-key `fuse-directory` is set to override this behaviour.
+
+### Filesystem attributes
+
+The filesystem attributes are sourced from the file obtained by `kdb file`.
+For some keys, no such file is returned, or the returned file does not exist in actuality. In these cases, fake attributes are used to preserve usability.
+
+To increase compatibility, currently the timestamps from these files are not used, as this prompts software like `vim` to produce warnings/errors.
+
+## Meta-Keys
+
+Meta-keys are mapped bidirectionally to extended file attributes (xattrs).
+To enable interoperability with other tools, the `meta:/` prefix is optional when reading/writing accessing meta keys. The listing of attributes however will show them prefix-less.
+
+Furthermore, no binary values for these attributes are allowed. (As libelektra does not accept those).
+
+See `xattr(1)` on how to read/write/list these attributes.
+
+## Binary-Keys
+
+If a value is to be written that cannot be decoded using the systems default encoding, it is treated as binary, and the appropriate `meta:/binary` meta-key is set.
+
+## Known issues
+
+- Moving an ``@elektra.value``-file corrupts the filesytsem/key database (See this [issue](https://github.com/ElektraInitiative/libelektra/issues/3648)). As editors like `vim` may move a file during editing, such editors cannot reliebly be used on these pseudo-files.
+- Only functional on UNIX-like systems
```

Comment:
Isn't the limitation simply that it will only work on systems that support FUSE?

```suggestion
- Only functional on Unix-like systems
```